modules/vault/src/test/java/org/testcontainers/vault/VaultClientTest.java

```diff
@@ -0,0 +1,58 @@
+package org.testcontainers.vault;
+
+import com.bettercloud.vault.Vault;
+import com.bettercloud.vault.VaultConfig;
+import com.bettercloud.vault.VaultException;
+import com.bettercloud.vault.response.LogicalResponse;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.testcontainers.containers.wait.strategy.Wait;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class VaultClientTest {
+
+    private static final int VAULT_PORT = 8200; //using non-default port to show other ports can be passed besides 8200
+
+    private static final String VAULT_TOKEN = "my-root-token";
+
+    @ClassRule
```

Comment:
1) could you please use try-with-resources
2) please also test the default constructor

Reply:
added